tests/generic_config_updater/test_pfcwd_interval.py

```diff
@@ -120,13 +120,49 @@ def get_new_interval(duthost, is_valid):
         return min(detection_time, restoration_time) + 10
 
 
+def test_stop_pfcwd(duthost, ensure_dut_readiness):
+    start_pfcwd = duthost.shell('config pfcwd start_default')
+    pytest_assert(not start_pfcwd['rc'], "Failed to start default pfcwd config")
+    pfcwd_config = duthost.shell("show pfcwd config")
+    pytest_assert(not pfcwd_config['rc'], "Unable to read pfcwd config")
+
+    for line in pfcwd_config['stdout_lines']:
+        if line.startswith('Ethernet'):
+            interface = line.split()[0]
+
+    json_patch = [
+        {
+            "op": "remove",
+            "path": "/PFC_WD/{}/detection_time".format(interface),
+        },
+        {
+            "op": "remove",
+            "path": "/PFC_WD/{}/restoration_time".format(interface),
+        },
+        {
+            "op": "remove",
+            "path": "/PFC_WD/{}/action".format(interface)
+        }
+    ]
+
+    try:
+        tmpfile = generate_tmpfile(duthost)
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+        pfcwd_updated_config = duthost.shell("show pfcwd config")
+        pytest_assert(not pfcwd_config['rc'], "Unable to read updated pfcwd config")
+        pytest_assert(interface not in pfcwd_updated_config['stdout'].split(), "pfcwd unexpectedly still running on interface {}".format(interface))
+    finally:
+        delete_tmpfile(duthost, tmpfile)
```

Comment:
this whole code needs to be indented. Needs to move under the 'for' block

Reply:
This test only intends to test one interface, as the flow would be the same for all interfaces. I added a break statement to the for loop to stop once the first interface is identified for test